### ISSUE TITLE
Named params

### ISSUE THREAD

**Issue by Althenar** [joeferner/node-oracle#218]
Is there something I'm missing or are named parameters in a query currently not supported? 

Ideally I'd love for execute() to accept a query with named params (:parName) and an object containing the values to substitute.

**Comment by b2dac** [comment]
I agree. This would be so great! Is this something that's being worked?

**Comment by ericwaldheim** [comment]
FYI: https://github.com/joeferner/node-oracle/issues/246#issuecomment-70934114

**Comment by b2dac** [comment]
thanks!
